openconfig_pyang/plugins/yangpath.py

```diff
@@ -175,19 +190,41 @@ def print_node(node, module, fd, prefix, ctx, level=0):
         print_children(node.i_children, module, fd, prefix, ctx, level)
 
 
-def get_pathstr(pathstr, config, ctx, level):
+def get_pathstr(pathstr, config, ctx, level, modname):
 
   if ctx.opts.print_plain or ctx.opts.relocate_output:
     return pathstr
 
   s = ''
-  if ctx.opts.print_depth:
-    s += '  [%d]' % level
-  else:
-    s += '     '
-  if config:
-    s += '  [%s] %s' % (config, pathstr)
+  if ctx.opts.prefix_module:
+    s += '%s:' % modname
```

Comment:
FYI, we have some guidance that says that we don't need this in the OpenConfig origin in gNMI (see https://github.com/openconfig/public/pull/542). I'm OK with having this here, but it's probably worth being aware of when we are specifying gNMI paths.

Reply:
It's not actually my intention to display this for the purpose of providing the origin namespace. Rather, the intent is to give information to the pyang user indicating which module the path belongs. I.e., to give a hint about where to look for more information about a given path. It's the same information that would otherwise be emitted by pyang, but formatted differently so that it's more convenient for grepping etc.

Reply:
ACK.